src/commons/sideContent/SideContentContestVotingContainer.tsx

```diff
@@ -19,30 +19,29 @@ type StateProps = {
 /**
  * Container to separate behaviour concerns from rendering concerns
  * Stores component-level voting ranking state
+ * Removed minScore and maxScore due to invalid scores no longer being possible
  */
 const SideContentContestVotingContainer: React.FunctionComponent<
   SideContentContestVotingContainerProps
 > = props => {
   const { canSave, contestEntries, handleSave, handleContestEntryClick } = props;
   const [isValid, setIsValid] = useState<boolean>(true);
   const [votingSubmission, setVotingSubmission] = useState<ContestEntry[]>([]);
-  const minScore = 11 - contestEntries.length;
-  const maxScore = 10;
+  // const minScore;
+  // const maxScore;
 
   useEffect(() => {
     setVotingSubmission(contestEntries);
   }, [contestEntries]);
 
   /**
-   * Validates input value and clamps the value within the min-max range [1, number of entries].
+   * Validates input value.
    * @param votingSubmission voting scores by user for each contest entry.
    * @returns boolean value for whether the scores are within the min-max range.
+   * Removed minScore and maxScore constraints.
    */
   const isSubmissionValid = (votingSubmission: ContestEntry[]) => {
-    return votingSubmission.reduce((isValid, vote) => {
-      const score = vote.score!;
-      return isValid && score >= minScore && score <= maxScore;
-    }, true);
+    return votingSubmission.reduce((isValid, vote) => isValid, true);
```

Comment:
This line of code seems like it would always return `true`. Is that the intended behaviour? If yes, can you explain the rationale? What's this part trying to do?